launcher/src/main/go/go.mod

```diff
@@ -0,0 +1,24 @@
+module launcher2
+
+go 1.21
```

Comment:
If we distribute the binaries and don't require downstream users to build this, can we require 1.23?